packages/matchbox/src/components/Table/tests/Table.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Table from '../Table';
+
+describe('Table', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    const props = {
+      title: 'Sample Table',
+      accent: true,
```

Comment:
Accent and title are not props

Will also need to test `children` pass through